middleware/locals.js

```diff
@@ -169,7 +169,11 @@ module.exports = {
             } else {
                 return false;
             }
-        };
+        }
+
+        /* Configure per-feature queries for use in templates */
+        res.locals.featureUseNewHeader = queryFeature('use-new-header');
+        res.locals.featurePreviewDigitalFund = queryFeature('preview-digital-fund');
```

Comment:
I probably would've put the `appData.isNotProduction` line here so you don't need to repeat it in templates

Reply:
Yeah I wondered about that – possibly there's a situation where we want a feature to be usable even in production?

Reply:
Putting that check here wouldn't prevent that use-case would it? We're just moving the check up.